src/lib.rs

```diff
@@ -191,21 +291,32 @@ pub mod pallet {
 		pub fn show_me_the_money(origin: OriginFor<T>) -> DispatchResultWithPostInfo {
 			let payee = ensure_signed(origin)?;
 
-			// Calculate the veted amount on demand.
+			// Calculate the vested amount on demand.
 			let mut info =
 				AccountsPayable::<T>::get(&payee).ok_or(Error::<T>::NoAssociatedClaim)?;
 			ensure!(
 				info.claimed_reward < info.total_reward,
 				Error::<T>::RewardsAlreadyClaimed
 			);
-			let now = frame_system::Pallet::<T>::block_number();
 
-			let payable_per_block = info.total_reward
+			ensure!(
+				info.total_reward > T::MinimumContribution::get(),
```

Comment:
I interpret `info.total_reward > T::MinimumContribution::get()` to mean "the total that is rewardable over the vesting period should be more than the min". Should this be "the currently rewardable should be greater than the min" instead?

Or am I perhaps misunderstanding the names here (I interpret `total_reward` to be the total amount payable over the vesting period)